samples/matter/lock/prj.conf

```diff
@@ -42,3 +42,6 @@ CONFIG_CHIP_APP_LOG_LEVEL=3
 # Enable Factory Data feature
 CONFIG_CHIP_FACTORY_DATA=y
 CONFIG_CHIP_FACTORY_DATA_BUILD=y
+
+# Set SUIT template paths
+CONFIG_SUIT_ENVELOPE_TEMPLATE="suit_templates/app_envelope.yaml.jinja2"
```

Comment:
surely this should be in an nrf54h20 overlay file?

Reply:
Right, moved to board overlay in this and subsequent occurrences.